### ISSUE TITLE
Contrast fixes

### ISSUE THREAD

**Issue by Cirkutry** [rose-pine/discord#31]
1.
<details>
<summary>Examples</summary>
<br>

![Image](https://github.com/user-attachments/assets/e0ce77f0-8d9d-46d4-a717-b66aec2d117a)

![Image](https://github.com/user-attachments/assets/0535ffe2-1854-4bae-95a4-95f466f6decc)

![Image](https://github.com/user-attachments/assets/0b863da5-5a11-41f0-b997-35a2481057c0)

![Image](https://github.com/user-attachments/assets/f6be2ece-b451-4339-8d35-c016668a2cc2)

![Image](https://github.com/user-attachments/assets/c25e7cb3-7c06-4efa-ba5c-3299f98ae3a7)

![Image](https://github.com/user-attachments/assets/d4019e02-7cdb-4100-8821-b094ad4f11e0)

![Image](https://github.com/user-attachments/assets/0a817cfb-11ed-4cda-8720-1eaa8229eb07)

![Image](https://github.com/user-attachments/assets/9d30f5ba-786f-4bcd-a0fa-f78cb97f0409)

</details>

Many areas (some examples above) where white text sits on the pink lacks contrast. It has a contrast ratio of only 2.91 which is low for all the small text. This will likely require quite a few changes but it's not a major gripe.

2.
![Image](https://github.com/user-attachments/assets/2c517ef2-71d8-4b23-891e-4e03265f80f1)
Muted channels are basically invisible.

**Comment by benji-en** [comment]
I believe changing the brand color to Pine help with this.

![Image](https://github.com/user-attachments/assets/ae1257cd-5ced-4e85-b6e3-1eed2601364a)
![Image](https://github.com/user-attachments/assets/92033144-a0fc-4c7d-9b59-ca1bf465ffde)
![Image](https://github.com/user-attachments/assets/8747b582-740a-4d3b-aead-6c8f00cb7394)
![Image](https://github.com/user-attachments/assets/4dfeb0a4-53b2-4b7e-95f8-fae7051868dd)
![Image](https://github.com/user-attachments/assets/8b83331b-d893-47b5-8ce0-bd116d02d879)
***
For this:

![Image](https://github.com/user-attachments/assets/b5690084-0597-424b-82f6-c898769ecd3a)

It is the same in the default Discord theme
***
Muted channels change: (I have muted channels hidden in most servers, so I missed this, sorry!)

![Image](https://github.com/user-attachments/assets/1a62e5ae-19cb-43ab-bcf6-96035179ff07)

**Comment by mvllow** [comment]
We use pine for buttons in other places, with white (text) foreground on the main variant. For dawn, however, we tend to use surface for the foreground.

As far as the muted channels, could those be the muted colour? They seem less visible than that but we should try to avoid going less contrast than our muted.

**Comment by benji-en** [comment]
Currently muted channels use `--interactive-muted`, which uses overlay currently, likely for other reasons.

This is what it looks like if it uses muted. IMO there's too little of a difference between a muted channel and one that isn't muted.
![Image](https://github.com/user-attachments/assets/3cb35b5d-67cd-4781-97e9-84b54f4ca240)

Muted categories use highlightMed, which could be a good pick.
![Image](https://github.com/user-attachments/assets/4ba6ed25-9404-43bd-b16e-55b8d2a288db)

**Comment by Cirkutry** [comment]
Wasn't able to figure out how to use those Pine shades with the current setup but I kinda like the pink shade that's going across the default theme. As I said it's not a major concern, more so an accessibility thing.

`highlightMed` seems a bit too dark and I personally prefer the `--interactive-muted` overlay for readability since channels names come into focus more than categories do.

**Comment by Cirkutry** [comment]
Ah, I misread oops. But yes `muted` is much more readable than what is being used currently and is also better than `highlightMed`.

**Comment by benji-en** [comment]
Okay, I'll change that to `muted`.

As for the brand color, I'm not sure if there's much I can do keep the `love` color being used currently and increase contrast without modifying the color quite a bit, so I think I will change it to `pine`.
